allennlp/nn/beam_search.py

```diff
@@ -78,7 +79,6 @@ def search(
         """
         Given a starting state and a step function, apply beam search to find the
         most likely target sequences.
-
```

Comment:
Why delete this line? I believe this blank line is needed for the docs to render properly.

Reply:
fix